### ISSUE TITLE
Ambassadors initiatives

### ISSUE THREAD

**Issue by saiyam1814** [cncf/ambassadors#89]
Like we have Books.md can we have something similar for Ambassadors initiatives and put guidelines around this 
Like I currently run 3 initiatives 
1 - youtube channel
2 - Challenges repository (in preparation for hactoberfest)
3 - Hubstation monthly newsletter 

Why another file? IMO all of the Ambassadors are working on some or the other good initiative which I feel should be a document so that we all can know each other's new initiatives. Now one Document can be a long if everyone keeps on adding, there can be a folder Ambassadors Initiatives with one .md file for each of the Ambassador where they can post their individual initiatives.
Initiatives are different from contributions for example Ambassador needs not add all the youtube videos one by one and they can give a youtube channel. Any OSS project they are working etc. LMK your thoughts on this if it makes sense to have this.

**Comment by chris-short** [comment]
What's the goal?

**Comment by saiyam1814** [comment]
The goal is to be aware of what all the Ambassadors are doing and maybe we can learn/contribute

**Comment by chris-short** [comment]
Interesting.

So the sake of discussion, let's assume we don't have any resources from CNCF initially.

What would the structure look like?

**Comment by saiyam1814** [comment]
Yes As of now we do not have anything from CNCF so maybe we can have a directory named - Ambassadors Initiatives and then Ambassadors can raise a PR with their name.md file where they will be putting their new initiatives.
Inside the .md file, there canbe few guidelines that can be defined like 
Inside Readme you can post new initiatives, new projects, new podcasts started, new newsletterstarted, new youtube channel started, new meetups started, new conferences started. etc.

**Comment by krishna-mk** [comment]
Looks interesting Saiyam...

IMO: 
Why don't create ONLY one file now and get all stuff in it? 
It also helps look at everything in a single location and good visibility of all activities.
If this file gets too big, then can split in various ways like active/inactive, yearly, or the one you mentioned individually as well.

**Comment by amye** [comment]
So we have the YouTube channel already, but a MD file in here seems like an easy start.

**Comment by saiyam1814** [comment]
Sure I will create a Sample File and create a PR
